lib/lexicon.py

```diff
@@ -18,11 +18,17 @@ class Lemma:
 
     def __init__(self, orth=None, phon=None, synt=None, eval=None, special=None):
         """
-        :param Optional[list[str]] orth:
-        :param Optional[list[str]] phon:
-        :param Optional[list[str]] synt:
-        :param Optional[list[str]] eval:
-        :param Optional[str] special:
+        :param Optional[list[str]] orth: list of spellings used in the training data
+        :param Optional[list[str]] phon: list of pronunciation variants. Each str should
+            contain a space separated string of phonemes from the phoneme-inventory.
+        :param Optional[list[list[str]]] synt: list of language model representations.
```

Comment:
I still have no idea why this needs to be a list of lists....

Reply:
alright, feel free to "insert a suggestion" for a docstring that you find more informative for the user.